src/Native/System.Native/CMakeLists.txt

```diff
@@ -31,8 +31,17 @@ add_library(System.Native
     ${NATIVE_SOURCES}
 )
 
+
+add_library(System.Native.Static
+    STATIC
+    ${NATIVE_SOURCES}
+)
+SET_TARGET_PROPERTIES(System.Native.Static PROPERTIES PREFIX "")
+SET_TARGET_PROPERTIES(System.Native.Static PROPERTIES OUTPUT_NAME System.Native CLEAN_DIRECT_OUTPUT 1)
```

Comment:
if you called it System.Native.a then the output file will be named System.Native.a.a as the build append the extension.